cfpb/sbl-frontend
# [Authentication] Route users to error page when Auth service is unreachable

Close #693 

## Changes

- Remove "Pre-MVP" from 500 page `Error details`
- Route users to 500 page when Authentication service is unreachable

## Considerations
- Is this a good place for the redirect? I needed a place inside the BrowserRouter (to support use of Navigate component) but didn't want to stick it in a component that was too specific so that this logic would apply to *any* authentication error.
- Should this be encapsulated in a custom hook so that we can apply this logic in a more targeted way?
- Can we simply label the `Error details` expandable as `Error details`? Does it *really* need qualifying (i.e. "for Pre-MVP..." or "for Beta")?

## How to test this PR

1. Stop Keycloak container
2. Visit unauthenticated homepage
3. Try to log-in
4. Observe that you are redirected to the `/500` page with `Error details` displayed

## Screenshots
https://github.com/cfpb/sbl-frontend/assets/2592907/26c5dc85-c00b-4088-a732-0165a81528bb
